test/Test/Management.hs

```diff
@@ -0,0 +1,111 @@
+        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
+          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
+          lExpectStorage baseDAO (checkPendingOwner (Just wallet1))
+          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet2)
+          lExpectStorage baseDAO (checkPendingOwner (Just wallet2))
+
+    , testCase "invalidates pending owner if new owner is current admin" $
+        integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
+          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
+          lExpectStorage baseDAO (checkPendingOwner (Just wallet1))
+          withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! testOwner)
+          lExpectStorage baseDAO (checkPendingOwner Nothing)
+    ]
+  , testGroup "Accept Ownership"
+      [ testCase "authenticates the sender" $
+          integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
+            withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
+            err <- expectError $ withSender wallet2 $ lCallEP baseDAO (Call @"Accept_ownership") ()
+            expectNotPendingOwner err
+
+      , testCase "changes the administrator to pending owner and resets pending owner" $
+          integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
+            withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
+            withSender wallet1 $ lCallEP baseDAO (Call @"Accept_ownership") ()
+            lExpectStorage baseDAO (checkAdministrator wallet1)
+            lExpectStorage baseDAO (checkPendingOwner Nothing)
+
+      , testCase "throws error when there is no pending owner" $
+          integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
+            err <- expectError $ withSender wallet1 $ lCallEP baseDAO (Call @"Accept_ownership") ()
+            expectNoPendingOwnerSet err
+
+      , testCase "throws error when called by current admin, when pending owner is not the address" $
+          integrationalTestExpectation $ withOriginated initialStorage $ \baseDAO -> do
+            withSender testOwner $ lCallEP baseDAO (Call @"Transfer_ownership") (#newOwner .! wallet1)
+            err <- expectError $ withSender testOwner $ lCallEP baseDAO (Call @"Accept_ownership") ()
+            expectNotPendingOwner err
+      ]
+  ]
+  where
+    initialStorage = mkStorage testOwner mempty mempty
+
+-- Helpers
+checkAdministrator :: Address -> Storage -> Either TestError ()
+checkAdministrator cadmin (sAdmin -> admin)
+  | admin == cadmin = pure ()
+  | otherwise =
+      Left $ CustomTestError $
+        "Administrator status was unexpected. Expected: " <>
+        (show cadmin) <> ", Found: " <> (show admin)
```

Comment:
1. Redundant brackets.
2. Why not `pretty` instead of `show`? `show @Address` doesn't look nice.

Reply:
This function no longer exists.